### ISSUE TITLE
Protocol mixed with Path when parsing Nginx access_log_combined

### ISSUE THREAD

**Issue by znarf** [sematext/logagent-js#72]
I have this line in my access log:

```127.0.0.1 - - [23/Jun/2017:16:25:15 +0200] "GET / HTTP/1.1" 200 612 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"```

I get the following object:

```
{ logSource: '/usr/local/var/log/nginx/access.log',
  _type: 'access_log_combined',
  client_ip: '127.0.0.1',
  remote_id: '-',
  user: '-',
  method: 'GET',
  path: '/ HTTP/1.1',
  status_code: 200,
  size: 612,
  referer: '-',
  user_agent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36',
  '@timestamp': 2017-06-23T14:25:15.000Z,
  message: 'GET / HTTP/1.1' }
```

Current: Path contains Protocol

Expected: Path without Protocol information, Protocol available in another field

**Comment by otisg** [comment]
Thanks @znarf 
@megastef - maybe https://github.com/sematext/logagent-js/blob/master/test/test.js#L56 needs to have a stricter check to catch this?

**Comment by megastef** [comment]
@znarf released fix on npm: v2.0.69

**Comment by znarf** [comment]
Thank you @megastef, it works!

However, I noticed that in patterns.yml, debug is set to true. Unintended I guess?

**Comment by megastef** [comment]
Thx. Will fix it.

**Comment by megastef** [comment]
Released 2.0.70 with debug:false in patterns.yml. 
https://github.com/sematext/logagent-js/commit/3de5a165134b52931a34936491f9a0014869a6f3
